pkgs/development/python-modules/mailman-rss/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, buildPythonPackage, fetchPypi
+, dateutil, future, requests, beautifulsoup4 }:
+
+buildPythonPackage rec {
```

Comment:
`mainmain-rss` looks like an application, so it should be in `all-packages.nix` and use `buildPythonApplication`.